simulariumio/mcell/mcell_converter.py

```diff
@@ -72,16 +72,20 @@ def _get_perpendicular_vector(v: np.ndarray, angle: float) -> np.ndarray:
         return McellConverter._rotate(u, v, angle)
 
     @staticmethod
-    def _get_rotation_matrix(v1: np.ndarray, v2: np.ndarray) -> np.matrix:
+    def _get_rotation_matrix(v1: np.ndarray, v2: np.ndarray) -> np.ndarray:
         """
         Orthonormalize and cross the vectors to get a rotation matrix
         """
         v1 = McellConverter._normalize(v1)
         v2 = McellConverter._normalize(v2)
         v2 = McellConverter._normalize(v2 - (np.dot(v1, v2) / np.dot(v1, v1)) * v1)
         v3 = np.cross(v2, v1)
-        return np.matrix(
-            [[v2[0], v1[0], v3[0]], [v2[1], v1[1], v3[1]], [v2[2], v1[2], v3[2]]]
+        return np.array(
```

Comment:
changes reviewed in [PR to this branch](https://github.com/simularium/simulariumio/pull/95)